integrationTests/p2p/peerDisconnecting/peerDisconnecting_test.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var durationBootstrapingTime = 2 * time.Second
+var durationBootstrapingTime = 5 * time.Second
```

Comment:
integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

Reply:
done